admin/app/controllers/admin/CommercialController.scala

```diff
@@ -31,8 +31,8 @@ object CommercialController extends Controller with Logging with AuthLogging wit
     NoCache(Ok(views.html.commercial.commercialMenu(environment.stage)))
   }
 
-  def renderCommercial = AuthActions.AuthActionTest { implicit request =>
-    NoCache(Ok(views.html.commercial.commercial(environment.stage)))
+  def renderCommercialRenderTimes = AuthActions.AuthActionTest { implicit request =>
+    NoCache(Ok(views.html.commercial.commercialRenderTimes(environment.stage)))
   }
```

Comment:
It looks broken, yes. I just renamed it to render times, to keep the urls tidy. We should look at fixing it, definitely.